utils/rpms/packaging/get_base_branch

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# find the base branch of the current branch
+
+set -eux -o pipefail
+
+max_commits=${1:-200}
+base_branch_re=${2:-'^  origin/(master|release/)'}
+
+n=0
+while [ "$n" -lt "$max_commits" ]; do
+    if git branch -r --contains HEAD~$n | grep -E "$base_branch_re" | sed -e 's/^ *[^\/]*\///'; then
+        exit 0
+    fi
+    ((n++)) || true
+done
```

Comment:
When we branch release/2.6 from master, during the short time that master == release/2.6, before we merge new commits into master, this will break because it will print two branches like
```
master
release/2.6
```

Reply:
Good catch.  I haven't pushed it yet but I have a different algorithm for determining this more along the lines of the one used in the git hooks.